### PR TITLE
Update to include app IDs

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

**Comment by robhudson** [comment]
Heads up to @hamilton @openjck @spasovski -- this PR will require both `product` and `app_id` (along with the usual dimensions) to be passed to the API for Fenix aggregation data.

File: glam/api/management/commands/import_glean_aggs.py [hunk 1]
```diff
@@ -11,7 +12,11 @@
 
 
 GCS_BUCKET = "glam-dev-bespoke-nonprod-dataops-mozgcp-net"
-PRODUCT_MODEL_MAP = {"org_mozilla_fenix": "api.FenixAggregation"}
+APP_TO_MODEL = {
+    "org_mozilla_fenix": "api.FenixAggregation",  # nightly
+    "org_mozilla_firefox_beta": "api.FenixAggregation",  # beta
+    "org_mozilla_firefox": "api.FenixAggregation",  # release
```
Comment (acmiyaguchi) [inline glam/api/management/commands/import_glean_aggs.py:18]: The `app_id`s in the probe scraper conventionally use kebab case: https://github.com/mozilla/probe-scraper/blob/master/repositories.yaml

**Review by acmiyaguchi — APPROVED** [review]
Looks reasonable to me, the only comment that the `app_id` is kebab cased in the probe scraper definition, and snake cased in the [etl scripts](https://github.com/mozilla/telemetry-airflow/blob/d13426a468f506567d9950a8c3635d08ed5db079/dags/glam_fenix.py#L19-L24). Being consistent with the ETL is probably the right choice here.

File: glam/api/management/commands/import_glean_aggs.py (hunk 1 above)
Comment (robhudson) [reply glam/api/management/commands/import_glean_aggs.py:18]: Thanks for pointing that out. The GLAM front-end will be passing this to the API, as well. I think the snake case is fine.

**Pull request merged**